notes/2023/2023-11-30.md

```diff
@@ -0,0 +1,43 @@
+# 2023-11-30 ESLint TSC Meeting Notes
+
+## Transcript
+
+[`2023-11-30-transcript.md`](2023-11-30-transcript.md)
+
+## Attending
+
+* Nicholas C. Zakas (@nzakas) - TSC
+* Milos Djermanovic (@mdjermanovic) - TSC
+
+@nzakas moderated, and @sam3k took notes.
+
+## Topics
+
+### [v9.0.0 Status](https://github.com/orgs/eslint/projects/4/views/2) 
+
+| Feature | Status | Notes |
+| ------- | -- | -- |
+| [Change Request: Drop Node v12/v14/v16/v17/v19 support](https://github.com/eslint/eslint/issues/17595) | PR is ready for merge | @nzakas and @mdjermanovic will set status to ready for merge |
+| [Change Request: Remove all formatters except stylish, html, json, and json-with-meta](https://github.com/eslint/eslint/issues/17524) | Waiting on the last remaining changes from @mdjermanovic | |
+| [Change default rule schema to `schema: []` and drop support for function-style rules](https://github.com/eslint/eslint/issues/14709) | @mdjermanovic opened [draft PR](https://github.com/eslint/eslint/pull/17792) | |
+| [Change Request: stop running rules set to "warn" when `--quiet` is used](https://github.com/eslint/eslint/issues/16450) | Close to done; just making final changes. | |
+| [Omiting any file/dir positional parameter to CLI does not give an error](https://github.com/eslint/eslint/issues/14308) | Ready for review | @mdjermanovic will review soon |
+| [Implement Language Plugins](https://github.com/eslint/eslint/issues/16999) | Once [#17698](https://github.com/eslint/eslint/pull/17698) and [#143](https://github.com/eslint-community/eslint-plugin-n/issues/143) are merged, we can move this one to ready to merge  | |
+| [Implement Flat Config](https://github.com/eslint/eslint/issues/13481) | **Blocked** | Blocked by work left on getting the rule tests switched over to use `FlatRuleTester`. Once that's merged, @nzakas will start on the docs. Also, we won't rename files in this PR. It will be handle separately. |
+| [Change Request: Remove deprecated context pass-through SourceCode methods](https://github.com/eslint/eslint/issues/17520) | This one will be ready along with the language plugins issue because it's all in the same PR | |
+| [Change Request: Enable `reportUnusedDisableDirectives` config option by default](https://github.com/eslint/eslint/issues/15466) | Waiting on @bmish to finish up the [PR (draft)](https://github.com/eslint/eslint/pull/17212) | |
+| | | |
+
+### [Contributor Pool for November](https://github.com/issues?q=org%3Aeslint+label%3A%22contributor+pool%22+merged%3A2023-11-01..2023-11-30)
```

Comment:
I believe you've told me to begin adding the contributor pool to this file? If not, let me know and I'll remove it.

Reply:
Yes, thank you!